cli/cmd/install/install.go

```diff
@@ -163,9 +163,12 @@ func (i *Install) preConfigure(ctx *clicontext.CLIContext, ignoreCluster bool) e
 	}
 
 	var totalMemory int64
+	versionWarning := false
 	for _, node := range nodes.Items {
 		totalMemory += node.Status.Capacity.Memory().Value()
-		checkKubernetesVersion(node.Status.NodeInfo.KubeletVersion)
```

Comment:
LGTM, can be simplied to save the version first. Then compare version outside the loop. Minor detail.